File: ClinicManagementSystem/ClinicManagement.Domain/Entities/Patient.cs
namespace ClinicManagement.Domain.Entities
{
    public class Patient
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public int Age { get; set; }
        public string Gender { get; set; } = string.Empty;
        public string Contact { get; set; } = string.Empty;
        public ICollection<Visit> Visits { get; set; } = new List<Visit>();
    }
}

File: ClinicManagement.AngularClient/src/app/models/patient.ts
export interface Patient {
  id: number;
  name: string;
  age: number;
  gender: string;
  contact: string;
  lastProblem: string;
  assignedDoctor: string;
  lastVisitDate?: string;
  visits?: Visit[];
}

export interface Doctor {
  id: number;
  name: string;
  specialization: string;
}

export interface Visit {
  id?: number;
  patientId: number;
  doctorId: number;
  problem: string;
  visitDate?: string;
  doctor?: Doctor;
}